integration_tests/pages/page.ts

```diff
@@ -71,6 +71,8 @@ export default abstract class Page {
 
   backLink = (): PageElement => cy.get('[data-test="back-link"]')
 
+  getMessages = (index: number): PageElement => cy.get('.moj-alert').eq(index)
```

Comment:
Adding this at the top `Page` level to have a consistent way to get messages on any sub-page.